src/index.ts

```diff
@@ -152,19 +234,32 @@ export default class OneSignal {
 
     /* L O C A T I O N */
 
-    static isLocationShared() {
+    /**
+     * True if the application has location share activated, false otherwise
+     * @returns Promise<boolean>
+     */
+    static isLocationShared(): Promise<boolean | void> {
```

Comment:
In the body, if the native module `RNOneSignal` is `null`, this method doesn't call native code and returns `Promise.resolve()` with no arguments, and because of that, I added `void` to the return type: 

> Type `Promise<void>` is not assignable to type `Promise<boolean>`.

That is incorrect? -> So I will now resolve the no-parameter `Promise` with a value instead.

Reply:
Done, resolve to `false`